src/Psl/Result/Ok.php

```diff
@@ -12,9 +12,9 @@
  *
  * @template    T
  *
- * @implements  IResultOrExceptionWrapper<T>
+ * @implements  IResult<T>
  */
-final class WrappedResult implements IResultOrExceptionWrapper
+final class Ok implements IResult
```

Comment:
Used the F# name here, couldn't come up with a better name myself.

Reply:
👍🏼

Reply:
What do you think about `Success`?  :thinking:

Reply:
Love it!